Added acknowledgement

premradhakrishnan opened the pull request.

leehawkinsssw commented:
Hi Prem. What content did you change on this rule? It looks you removed Adam and added yourself in the Acknowledgements, why?

premradhakrishnan commented:
> Hi Prem. What content did you change on this rule? It looks you removed Adam and added yourself in the Acknowledgements, why?

Hi Lee, I had updated the content a while back but had forgotten to add myself in the acknowledgements. I don't think I have removed Adam. I have added myself as another author.
